src/components/PDFView/PDFPasswordForm.js

```diff
@@ -132,6 +132,7 @@ function PDFPasswordForm({isFocused, isPasswordInvalid, shouldShowLoadingIndicat
                 secureTextEntry
             />
             <Button
+                onMouseDown={(e) => e.preventDefault()}
```

Comment:
Please add a short comment mentioning which default behavior we're preventing and why